src/site/content/en/handbook/content-checklist/index.md

```diff
@@ -27,19 +27,19 @@ This is a checklist for web.dev content reviews. It includes critical high-level
 ## Instruction
 1. If the piece expects readers to have prior knowledge, is that made clear up front?
 1. Does the piece explain why the process being taught matters?
-1. Are examples provided to support all key claims and ideas?
-1. Are instructions in second person (_you/your_), not first person (_we/our_)?
-1. Are common instructions provided using the web.dev [Instruction components](/handbook/web-dev-components/#instruction)?
+1. Do the examples support all key claims and ideas?
+1. Are instructions written in second person (_you/your_), not first person (_we/our_)?
+1. Do the common instructions follow the web.dev [Instruction components](/handbook/web-dev-components/#instruction)?
 
 ## Media
-1. Are images and videos used to clarify ideas that would be difficult to understand from text alone?
-1. Are any images or videos included that don't directly relate to ideas in the text?
+1. Do the images and videos clarify ideas that would be difficult to understand from text alone?
```

Comment:
Similar to the examples question above, this changes the meaning. I'd prefer to revert so it's clear we're asking whether people are including images and video where they're needed--though improved phrasing welcome!

Reply:
I agree with Michael on this. Someone actually violated item two recently. You wouldn't think we'd need to say it, but alas.